validator/js/engine/parse-srcset.js

```diff
@@ -69,22 +69,22 @@ const parseSrcset = function(srcset) {
   // more spaces + a non empty string containing no space or commas).
   // Doesn't capture the initial space.
   //
-  // \s*                       Match, but don't capture leading spaces
-  // (?:,\s*)?                 Optionally match comma and trailing space,
-  //                           but don't capture comma.
-  // ([^,\s]\S*[^,\s])         Match something like "google.com/favicon.ico"
-  //                           but not ",google.com/favicon.ico,"
-  // \s*                       Match, but dont capture spaces.
-  // ([\d]+.?[\d]*[w|x])?      e.g. "5w" or "5x" or "10.2x"
-  // \s*                       Match, but don't capture space
-  // (?:(,)\s*)?               Optionally match comma and trailing space,
-  //                           capturing comma.
+  // \s*                           Match, but don't capture leading spaces
+  // (?:,\s*)?                     Optionally match comma and trailing space,
+  //                               but don't capture comma.
+  // ([^,\s]\S*[^,\s])             Match something like "google.com/favicon.ico"
+  //                               but not ",google.com/favicon.ico,"
+  // \s*                           Match, but dont capture spaces.
+  // ([\\d]+(?:\\.[\\d]+)?[w|x])?  e.g. "5w" or "5x" or "10.2x"
```

Comment:
Please unescape these. Eg, it should just be `\d`, not `\\d`.

Reply:
Ah, right, I just copy-pasted the section in there, I'll change it.

Reply:
Fixed via https://github.com/ampproject/amphtml/pull/30252/commits/5fe27db958bab8f4b8cdee83258304ea4bb797bd